<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeCCProperty</class>
 <widget class="QWidget" name="ChangeCCProperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0" colspan="2">
    <widget class="KeyBoard" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string> 更改车次属性</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>车次号</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>机车速率</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>机车号</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>电力牵引</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KeyBoard</class>
   <extends>QWidget</extends>
   <header>menuclass/LeadSealing/keyboard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
